src/staticmaps.js

```diff
@@ -414,6 +414,18 @@ class StaticMaps {
    */
   getTile(data) {
     return new Promise((resolve, reject) => {
+      // custom for rendering map without base layer
```

Comment:
Might be nice to add a small test case for it

Reply:
On this repo or on api?

Reply:
This one

Reply:
won't do as tests don't run as discussed